packages/SystemUI/src/org/codeaurora/internal/IDepersoResCallback.aidl

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2015, The Linux Foundation. All rights reserved.
```

Comment:
Where are these added from? Seem to be from CAF, if so please add the commits with proper authorship